tests/bench.py

```diff
@@ -130,3 +130,10 @@ def run_compute():
     assert div_results["Frame Buffer 0"]["False Negative Divisions"] == 25
     assert div_results["Frame Buffer 0"]["False Positive Divisions"] == 31
     assert div_results["Frame Buffer 0"]["True Positive Divisions"] == 69
+
+
+def test_track_overlap_metrics(benchmark, iou_matched):
+    def run_compute():
+        return TrackOverlapMetrics(copy.deepcopy(iou_matched)).compute()
```

Comment:
Do we still need these `deepcopy` calls with the new Matched objects? Yes, right, because we are reusing the Matched object across benchmarks and don't want the order that they run to affect the benchmark?

Reply:
That was the original logic. Since it's a fixture, I was paranoid about cross-contaminating across the different benchmarking functions.